packages/bits/src/lib/date-picker/date-picker.component.ts

```diff
@@ -302,6 +302,10 @@ export class DatePickerComponent implements OnChanges, OnInit, ControlValueAcces
             this.calendarChanged.unsubscribe();
         }
 
+        if (this.overlay.showing) {
+            this.overlay.hide();
+        }
```

Comment:
it would be good to add a unit test or some form of automation test for this

Reply:
Will add, thanks